demos/system_reference-NXP.rst

```diff
@@ -6,11 +6,116 @@
 NXP Reference Boards
 ====================
 
-A number of the
-`OpenAMP project examples <https://github.com/OpenAMP/openamp-system-reference/tree/main/examples>`_
+A number of the `OpenAMP project examples
+<https://github.com/OpenAMP/openamp-system-reference/tree/main/examples>`_
 can be executed on NXP Reference boards.
 
-* NXP `8MPLUSLPD4-EVK <https://www.nxp.com/design/design-center/development-boards-and-designs/8MPLUSLPD4-EVK>`_.
+Board setup
+------------
 
-The `AN13970 Running Zephyr RTOS on Cadence Tensilica HiFi 4 DSP <https://www.nxp.com/docs/en/application-note/AN13970.pdf>`_
-application note provides instructions for setting up the evaluation board.
+The `i.MX 8M Plus Evaluation Kit Quick Start Guide (IMX8MPLUSQSG)
+<https://www.nxp.com/docs/en/quick-reference-guide/8MPLUSEVKQSG.pdf>`_
+contains basic information on the board and how to
+set it up.
+In `AN13970 Running Zephyr RTOS on Cadence Tensilica HiFi 4 DSP
+<https://www.nxp.com/docs/en/application-note/AN13970.pdf>`_
+application note provides instructions on how to launch the samples on
+the HiFi4 DSP.
+
+Generate the Zephyr rpmsg multi service example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Initialize the Zephyr environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Refer to `zephyr example readme
+<https://github.com/OpenAMP/openamp-system-reference/blob/main/examples/zephyr/README.md#initialization>`_
+article.
+
+Build the Zephyr image
+^^^^^^^^^^^^^^^^^^^^^^
+
+From the openamp-system-reference directory
+
+.. code-block:: console
+
+   west build -p -b imx8mp_evk//adsp -s examples/zephyr/rpmsg_multi_services/
+
+For details refer to `rpmsg_multi_services readme
+<https://github.com/OpenAMP/openamp-system-reference/blob/main/examples/zephyr/rpmsg_multi_services/README.rst#building-the-application>`_ article.
+
```

Comment:
Did you try to replace by following?
```
For details refer to :ref: `rpmsg_multi_services readme<openAMP_rsc_table_sample>`
```

`openAMP_rsc_table_sample` is the reference label of the page

Reply:
syntax is not good 
you can not specify `#building-the-application`
see https://www.sphinx-doc.org/en/master/usage/referencing.html for details

regarding the result seems that you also need to remove the space after :ref:
```
For details refer to :ref:`rpmsg_multi_services readme<openAMP_rsc_table_sample>`
```

Reply:
yes ! only link to `https://github.com/OpenAMP/openamp-system-reference/blob/main/examples/zephyr/README.md` has not be resolved using same kind of reference. 
It seems that we should consider replacing Markdown with reStructuredText format in the future to generate the documentation

Anyway if you are ok with this PR you can merge it